rfcs/0000-dns-cluster-aware-delegation.md

```diff
@@ -79,35 +82,70 @@ type: Opaque
 
 ## Namespace requirements for multi cluster
 
-ToDo Expectations of layout of namespaces and names of DNSRecords to match across clusters
+ToDo mnairn: Expectations of layout of namespaces and names of DNSRecords to match across clusters
+
+## Default Provider Secret
+
+A new concept of "default provider secret" will be introduced that removes the need for a `providerRef` to be specified on every DNSRecord and DNSPolicy.
+When a DNSPolicy is created with no `providerRef` the resulting DNSRecord during initial reconciliation will search for a default provider secret in the current namespace to use instead.
+The DNSRecord status will be updated to include the selected provider secret as a `providerRef` in the status.
+A default provider secret is any provider secret that has been appropriately labelled with the kuadrant default provider label (kuadrant.io/default-provider=true)
+Only one default provider can exist in any namespace and more than one labelled as such will result in an error (mnairn: I have actually no idea where????) 
```

Comment:
It should populate onto the status of the DNSRecord IMO which will then also show on the DNSPolicy